### DISCUSSION TITLE
Use case for non-platform users

### DISCUSSION THREAD

**Discussion by raphattack** [photoprism/photoprism#3430] [Q&A]
Hey all, thinking about giving Photoprism a shot, but wanted to ask the community if it currently supports these use cases:

1. Share album with non-platform users
2. Allow non-platform users to comment
3. Allow non-platform users to add to album

My use case is that it will be very hard to get my friends/family to migrate over to a new platform, but we all take pictures at events together and add everything into a shared album.

Is it possible to do any of the above if I (a Photoprism user) share an album with my friends/family (who are non-Photoprism users)?

**Comment by lastzero** [comment]
2 and 3 are not implemented yet, but planned. Unfortunately, we can't give an exact release date due to the number of feature requests on our roadmap.
